operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -543,13 +543,24 @@ public static boolean isMaintenanceTimeWindowsSatisfied(Reconciliation reconcili
     }
 
     /**
-     * Encodes a String into Base64
+     * Encodes a String into Base64.
      *
-     * @param encode    String which should be encoded
+     * @param data String that should be encoded.
      *
-     * @return          Base64 data
+     * @return Base64 data.
      */
```

Comment:
We tend to keep the Javadoc comments with tabs normally. So no need to change this I guess?